scipy/sparse/csgraph/tests/test_pydata_sparse.py

```diff
@@ -149,6 +150,10 @@ def test_min_weight_full_bipartite_matching(graphs):
     assert_equal(actual, desired)
 
 
+@pytest.mark.skipif(
+    _pep440.parse(sparse.__version__) < _pep440.Version("0.15.4"),
```

Comment:
does this behave correctly when `sparse is None`?

Reply:
Right, I updated it to also take into account missing `sparse`.